app/src/lib/git/core.ts

```diff
@@ -161,7 +161,7 @@ export async function git(
     // from a terminal or if the system environment variables
     // have TERM set Git won't consider us as a smart terminal.
     // See https://github.com/git/git/blob/a7312d1a2/editor.c#L11-L15
-    opts.env = { TERM: 'dumb', ...combinedEnv } as Object
+    opts.env = { TERM: 'dumb', ...combinedEnv } as object
```

Comment:
Note -- 99% of the time, `object` is the type you want. `Object` is effectively a TS implementation detail that shouldn't be used in user code